src/components/appointments/AppointmentForm.js

```diff
@@ -0,0 +1,183 @@
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {Route, Switch, useHistory, useRouteMatch} from 'react-router-dom';
+
+import {ConfigContext} from 'Context';
+import Body from 'components/Body';
+import Card from 'components/Card';
+import ErrorBoundary from 'components/ErrorBoundary';
+import FormDisplay from 'components/FormDisplay';
+import {LiteralsProvider} from 'components/Literal';
+import Loader from 'components/Loader';
+import ProgressIndicatorDisplay from 'components/ProgressIndicator/ProgressIndicatorDisplay';
+import {STEP_LABELS} from 'components/ProgressIndicator/constants';
+import SummaryConfirmation from 'components/SummaryConfirmation';
+import AppointmentStep from 'components/appointments/AppointmentStep';
+import {SUBMISSION_ALLOWED} from 'components/constants';
+import useTitle from 'hooks/useTitle';
+
+const AppointmentProgressIndicator = ({title}) => {
+  const config = useContext(ConfigContext);
+  const summaryMatch = !!useRouteMatch('/appointment/overzicht');
+  const confirmationMatch = !!useRouteMatch('/appointment/bevestiging');
+  const appointmentMatch = !!useRouteMatch('/appointment');
+
+  // figure out the title for the mobile menu based on the state
+  let activeStepTitle;
+  if (summaryMatch) {
+    activeStepTitle = STEP_LABELS.overview;
+  } else if (confirmationMatch) {
+    activeStepTitle = STEP_LABELS.confirmation;
+  } else {
+    activeStepTitle = STEP_LABELS.appointment;
+  }
+
+  const ProgressIndicatorDisplayComponent =
+    config?.displayComponents?.progressIndicator ?? ProgressIndicatorDisplay;
+
+  return (
+    <ProgressIndicatorDisplayComponent
+      activeStepTitle={activeStepTitle}
+      formTitle={title}
+      steps={[]}
+      hasSubmission={false}
+      isStartPage={false}
+      isSummary={summaryMatch}
+      isConfirmation={confirmationMatch}
+      isAppointment={appointmentMatch}
+      isSubmissionComplete={false}
+      areApplicableStepsCompleted={false}
+      showOverview={true}
+      showConfirmation={true}
+      showAppointment={true}
+    />
+  );
+};
+
+const AppointmentSummary = ({form, onConfirm}) => {
+  const history = useHistory();
+  const onPrevPage = event => {
+    event.preventDefault();
+    history.push('/appointment');
+  };
+
+  const intl = useIntl();
+  const pageTitle = intl.formatMessage({
+    description: 'Summary page title',
+    defaultMessage: 'Check and confirm',
+  });
+  useTitle(pageTitle);
+
+  const [privacy, setPrivacy] = useState({
+    requiresPrivacyConsent: true,
+    privacyLabel: '',
+    policyAccepted: false,
+  });
```

Comment:
seeing the repetition with existing code, this probably means we need a privacy row/component to encapsulate the state/details inside.

Reply:
Should we make it in the next PRs? It's the same as with Progress indicator - I want to make changes which affect the existing process in separate PRs with additional care